DOC-10325 index missing leading key (#2871)
couchbase/docs-server: modules/n1ql/pages/n1ql-language-reference/createindex.adoc

index-key:: [Required] Specifies an index key.
Refer to <<index-key>> below.

> 

Reviewer: Not sure why this vertical space is here?

```suggestion
```

* If this clause is present, the statement does nothing and completes without error.

[[include-missing]]
=== INCLUDE MISSING Clause
[.status]#Couchbase Server 7.1.2#

> Prior to version 7.1.2, if the field denoted by `index-key` does not exist in the document,
> then the document was not indexed.

Reviewer: Slight inconsistency of tenses

```suggestion
In Couchbase Server versions prior to 7.1.2, if the field denoted by `index-key` does not exist in the document,
then the document is not indexed.
```

> For versions 7.1.2{plus}, the `INCLUDE MISSING` clause will ensure that documents which do not hold the `index-key` field are indexed regardless.
> Documents without the `index-key` field will not be indexed
> (The same behaviour for Couchbase Server versions prior to 7.1.2)

Reviewer: Clarification

```suggestion
For Couchbase Server version 7.1.2 or later, the optional `INCLUDE MISSING` clause ensures that documents which do not hold the `index-key` field are indexed regardless.
If this clause is not present, then documents without the `index-key` field are not indexed.
```

> NOTE: The `INCLUDE` part of the clause can be left out.

Reviewer: Clarification

```suggestion
The `INCLUDE` keyword is optional.
Omitting this keyword makes no difference to the operation of the `INCLUDE MISSING` clause.
```
